OpenRA.Game/Server/Server.cs

```diff
@@ -154,16 +154,14 @@ public Server(IPEndPoint endpoint, ServerSettings settings, ModData modData)
 				}
 			};
 
-			FieldLoader.Load(LobbyInfo.GlobalSettings, modData.Manifest.LobbyDefaults);
-
-			foreach (var t in serverTraits.WithInterface<INotifyServerStart>())
-				t.ServerStarted(this);
-
-			Log.Write("server", "Initial mod: {0}", ModData.Manifest.Mod.Id);
-			Log.Write("server", "Initial map: {0}", LobbyInfo.GlobalSettings.Map);
-
 			new Thread(_ =>
 			{
+				foreach (var t in serverTraits.WithInterface<INotifyServerStart>())
+					t.ServerStarted(this);
```

Comment:
That was actually the entire purpose of this change: our horrible load screen code will otherwise flash the load screen when opening the skirmish lobby (because it is running on the main thread).  Fixing the jank when starting a skirmish was just a nice side effect.

As far as I can tell this should be safe: we already load maps off the main thread in the lobby and mission browser, and the server does not start listening for clients until later in the thread.